.env.default

```diff
@@ -7,3 +7,12 @@ FLASK_ENV=development
 MYPYPATH=mypy_stubs
 
 SENTRY_DSN=
+
+# So urls can be routed consistently internally
+SERVER_NAME=http://localhost:5000
+
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
+
+# Will be set automatically by the server
+FLASK_SECRET_KEY=
```

Comment:
The main reason is that it persists between server start/stops, so the session of a user would be preserved. This wouldn't really matter in production, as it doesn't really restart, but for local dev it makes things easier. The production instance should still have access to the basic filesystem, so it should still work.

Reply:
I believe we discussed/resolved this: we'll leave this value here so that local builds can make use of it, but in production we will generate our own random token and use Heroku to push it to the app.

In general, can we follow that procedure with all of the remaining values in this file? In particular, the `GOOGLE_CLIENT_*` values should all be pushed via Heroku, and we would possibly copy those to a local .env setup if we needed the login functionality?

Reply:
Yes, you can just put those into your local `.env` file if you need them (which is what I've done throughout this entire PR).

Reply:
We need to track the task of pushing these variables to Heroku, and also we need to document them somewhere. Could you open an issue listing the variables that must be set by us in production? We can make sure the deploy has them, and that we fix up our docs in the same stroke.

Reply:
#371